libraries/matware/jupgrade/schemas/1.5/contents.php

```diff
@@ -179,8 +179,8 @@ public function dataHook($rows = null)
 			// @@ Duplicate entry for key 'idx_client_id_parent_id_alias_language'
 			if ($content->load(array('alias' => $row['alias'], 'catid' => $row['catid'])))
 			{
-				// Getting the duplicated alias
-				$alias = $this->getAlias('#__content', $row['alias']);
+				$content->reset();
+				$content->id = 0;
 				// Set the modified alias
 				$row['alias'] .= "-".rand(0, 99999999);
```

Comment:
this line
    $content->load(array('alias' => $row['alias'], 'catid' => $row['catid']))
searches for item by using alias and catid, **but it also loads it** if there is already an item with such alias in this category (which is intended as it was). So in this case  you have loaded this item into content JTableContent instance. This fact changes the way 
    $content->bind($row)
will do its work, **instead of creating new item** from $row data as it is happens when there is no item with same alias, **it will merge $row with the item you loaded** with $content->load and update it, while **id from $row is ignored**.

This is why we need to reset JTableContent instance and set its id to 0 - this reverts changes done by $content->load to this instance.